content/en/synthetics/ci.md

```diff
@@ -223,7 +278,61 @@ The configuration of synthetics test occurs in `**/*.synthetics.json` (configura
+     * **blocking**: The CLI returns an error if the test fails.
+     * **non_blocking**: The CLI only prints a warning if the test fails.
+     * **skipped**: The test is not executed at all.
+* **startUrl**: (_string_) New start URL to provide to the test.
+* **variables**: (_object_) Variables to replace in the test. This object should contain, as keys, the name of the variable to replace and, as values, the new value of the variable.
+
+#### Execution rule
+
+The _execution rule_ of each test can also be defined in-app, at the test level. Use the drop-down menu next to **CI Execution**.
+
+{{< img src="synthetics/ci/execution_rule.mp4" alt="CI Execution Rule" video="true" width="100%">}}
+
+The execution rule associated with the test is always the most restrictive one that was set in the configuration file. From the most restrictive to the least restrictive: `skipped`, `non_blocking`, `blocking`. For example, if your test is configured to be `skipped` in the UI but to `blocking` in the configuration file, it is `skipped` when running your tests. 
+
+#### Start URL
+
+You can configure on which url your test starts by providing a `startUrl` to your test object and build your own starting url using any part of your test's original starting url and the following environment variables:
+
+| Environment variable | Description                  | Example                                                |
+|----------------------|------------------------------|--------------------------------------------------------|
+| `URL`                | Test's original starting url | `https://www.example.org:81/path/to/something?abc=123` |
+| `DOMAIN`             | Test's domain name           | `example.org`                                          |
+| `HOST`               | Test's host                  | `www.example.org:81`                                   |
+| `HOSTNAME`           | Test's hostname              | `www.example.org`                                      |
+| `ORIGIN`             | Test's origin                | `https://www.example.org:81`                           |
+| `PARAMS`             | Test's query parameters      | `?abc=123`                                             |
+| `PATHNAME`           | Test's URl path              | `/path/to/something`                                   |
+| `PORT`               | Test's host port             | `81`                                                   |
+| `PROTOCOL`           | Test's protocol              | `https:`                                               |
+| `SUBDOMAIN`          | Test's sub domain            | `www`                                                  |
+
+For instance, if your test's starting url is `https://www.example.org:81/path/to/something?abc=123`, it can be written as:
+
+* `{{PROTOCOL}}//{{SUBDOMAIN}}.{{DOMAIN}}:{{PORT}}{{PATHNAME}}{{PARAMS}}`
+* `{{PROTOCOL}}//{{HOST}}{{PATHNAME}}{{PARAMS}}`
+* `{{URL}}`
```

Comment:
Would be nice to give all examples with overrides since it's the purpose of that feature, like `http://localhost:8080{{PATHNAME}}{{PARAMS}}` to change domain url, or `{{PROTOCOL}}//{{SUBDOMAIN}}.{{DOMAIN}}:8081{{PATHNAME}}{{PARAMS}}` to change only the port.